client/rpc/src/lib.rs

```diff
@@ -134,7 +162,9 @@ where
         // https://doc.rust-lang.org/std/io/trait.Read.html#errors-1
         loop {
             let mut chunk = Vec::with_capacity(FILE_CHUNK_SIZE as usize);
-            let read_result = file.by_ref().take(FILE_CHUNK_SIZE).read_to_end(&mut chunk);
```

Comment:
Why do we need this change? Just trying to understand

Reply:
`by_ref` in this case is a method part of the `std::io::Read` trait.
Prior to this we only had in scope this trait, but now we also have `std::io::Write` which also has a method `by_ref`. This rightfully confuses the compiler.

Reply:
Makes sense!